frontend/src/bundles/auth/components/protected-route/protected-route.tsx

```diff
@@ -0,0 +1,22 @@
+import { type FC, type ReactNode } from 'react';
+import { Navigate } from 'react-router-dom';
+
+import { AppRoute } from '~/bundles/common/enums/app-route.enum.js';
+
+type ProtectedRouteProperties = {
+    children: ReactNode;
+};
```

Comment:
We name this type Properties in other component, please rename it to keep the same naming in whole project
```suggestion
type Properties = {
    children: ReactNode;
};
```